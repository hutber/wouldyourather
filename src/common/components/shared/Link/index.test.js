import React from 'react'
import Component from './'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Add New button', () => {
	it('renders add new button properly', () => {
		const tree = shallow(<Component to="/home" text="You can add 2 more children" />)
		expect(toJson(tree)).toMatchSnapshot()
	})
})
