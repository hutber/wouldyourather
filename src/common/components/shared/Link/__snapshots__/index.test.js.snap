// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add New button renders add new button properly 1`] = `
<GuttersGrid
  classes={
    Object {
      "content": "GuttersGrid-content-1",
    }
  }
  text="You can add 2 more children"
  to="/home"
/>
`;
